internal/controller/managedzone_controller.go

```diff
@@ -398,6 +408,21 @@ func (r *ManagedZoneReconciler) parentZoneNSRecordReady(ctx context.Context, man
 	return nil
 }
 
+func (r *ManagedZoneReconciler) deleteOwnedDNSRecords(ctx context.Context, managedZone *v1alpha1.ManagedZone) (int, error) {
```

Comment:
question: this method is returning the `remainingRecordsCount` or the number of dns record in the namespace?

Reply:
ooh good catch, thanks